content/it/docs/concepts/architecture/control-plane-node-communication.md

```diff
@@ -0,0 +1,67 @@
+
+
+<!-- body -->
+
+## Dal Nodo al control-plane{#dal-nodo-al-control-plane}
+Kubernetes adotta un pattern per le API di tipo _"hub-and-spoke"_. Tutte le chiamate delle API eseguite sui vari nodi sono effettuate verso l'apiserver (nessuno degli altri componenti principali è progettato per esporre servizi remoti). L'apiserver è configurato per l'ascolto di connessioni remote su una porta HTTPS protetta (443) con una o più forme di [autenticazioni client](/docs/reference/access-authn-authz/authentication/) abilitate. Si dovrebbero abilitare una o più forme di [autorizzazioni](/docs/reference/access-authn-authz/authorization/), in particolare nel caso in cui siano ammesse [richieste anonime](/docs/reference/access-authn-authz/authentication/#anonymous-requests) o [_token_ legati ad un account di servizio (_service account_)](/docs/reference/access-authn-authz/authentication/#service-account-tokens).
+
+Il certificato pubblico (_public root certificate_) relativo al cluster corrente deve essere fornito ai vari nodi di modo che questi possano connettersi in modo sicuro all'apiserver insieme alle credenziali valide per uno specifico _client_. Ad esempio, nella configurazione predefinita di un cluster [GKE](https://cloud.google.com/kubernetes-engine?hl=it), le credenziali del client fornite al kubelet hanno la forma di un certificato client. Si veda
+[inizializzazione TLS del kubelet TLS](/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping/) per la fornitura automatica dei certificati client al _kubelet_.
+
+I _pod_ che desiderano connettersi all'apiserver possono farlo in modo sicuro sfruttando un account di servizio in modo che Kubernetes inserisca automaticamente il certificato pubblico di radice e un token valido al portatore (_bearer token_) all'interno _pod_ quando questo viene istanziato.
+In tutti i namespace è configurato un _Service_ con nome `kubernetes` con un indirizzo IP virtuale che viene reindirizzato (tramite _kube-proxy_) all'endpoint HTTPS dell'apiserver.
+
+Anche i componenti del piano d controllo comunicano con l'apiserver del cluster su di una porta sicura esposta da quest'ultimo.
+
+Di conseguenza, la modalità operativa predefinita per le connessioni dai nodi e dai _pod_ in esecuzione sui nodi verso il _control-plane_ è protetta da un'impostazione predefinita
+e può essere eseguita su reti non sicure e/o pubbliche.
+
+## Dal control-plane al nodo{#dal-control-plane-al-nodo}
+
+Esistono due percorsi di comunicazione principali dal _control-plane_ (apiserver) verso i nodi. Il primo è dall'apiserver verso il processo _kubelet_ in esecuzione su ogni nodo nel cluster. Il secondo è dall'apiserver a ciascun nodo, _pod_, o servizio attraverso la funzionalità proxy dell'apiserver.
+
+### Dall'apiserver al _kubelet_
+
+Le connessioni dall'apiserver al _kubelet_ vengono utilizzate per:
+
+  * Prendere i log relativi ai vari _pod_.
+  * Collegarsi (attraverso kubectl) ai pod in esecuzione.
```

Comment:
nit (OK to skip fixing this, but nice to have):
```suggestion
  * Prendere i log relativi ai vari _Pod_.
  * Collegarsi (attraverso kubectl) ai Pod in esecuzione.
```
because pod is not a common word in Italian and, here, clearly refers to the API object.

Reply:
ok I will change it according the rule we agreed above for k8s resources

Reply:
just a not... if the word refers to k8s resource will have first letter uppercase but do I have to skip the italic in this case?